src/main/scala/cromwell/binding/types/WdlObjectType.scala

```diff
@@ -1,16 +1,30 @@
 package cromwell.binding.types
 
 import cromwell.binding.Call
-import cromwell.binding.values.{WdlCallOutputsObject, WdlObject}
+import cromwell.binding.values.{WdlCallOutputsObject, WdlMap, WdlObject, WdlString}
+
+import scala.util.{Failure, Success}
 
 case object WdlObjectType extends WdlType {
   val toWdlString: String = "Object"
 
   override protected def coercion = {
     case o: WdlObject => o
+    case m: WdlMap if m.wdlType.keyType == WdlStringType && WdlStringType.isCoerceableFrom(m.wdlType.valueType) =>
```

Comment:
I'm not sure I understand why the coercion from `Map[String, String]` to `Object` exists.  Is this used anywhere or added because it might be useful?

Also, since this chunk is used more than once (also in `isCoerceableFrom`, perhaps extract it?

```scala
m.wdlType.keyType == WdlStringType && WdlStringType.isCoerceableFrom(m.wdlType.valueType)
```

I wonder if the key can be coercable to string like the value:

```scala
WdlStringType.isCoerceableFrom(m.wdlType.keyType) && WdlStringType.isCoerceableFrom(m.wdlType.valueType)
```